workshops/python and data visualizations/README.md

```diff
@@ -0,0 +1,325 @@
+
+-   For example in our code so far we have used…
+```python
+fig,ax= plt.subplots()
+```
+     
+- This created a figure with a single axis.
+- Here are a few different ways you can use fig…
+```python
+fig=plt.figure() 
+#creates empty graph returns with num. figues
+fig,ax =plt.subplots(2,2) #creates 2 graph figues
+```
+Check it out in your Jupyter Editor!
+
+*Can you tell I have a strange fascination with random things dancing hehe!*
+
+## Step Three!
+
+*Make sure libraries are uploaded, look back @ Step Two if needed*.
+
+Set up spacing of your x and y axis...
+ ```python
+ x=np.linspace(0,2,100)
+ ```
+
+\*Make sure your libraries are uploaded!
+
+## Step Four!
+
+Create a figure! Scroll back up to see how to write that code...
+Here is a hint 
+
+`fig,ax= plt._____()`
+
+*Still stuck? Where the underscores are you should put*
+
+`fig,ax= plt.subplots()`
+
+## Step Five!
+
+Now we can plot some lines, in this part customize it however you want!
+
+This example is going to have 3 basic functions (linear, squared, and cubed) but you can fancy yours up any way you want!
+
+```python 
+ax.plot(x,x,label='Sleep')
+ax.plot(x,x**2,label='Worry if sky is falling...')
+ax.plot(x,x**3,label='Be a icon')
+```
+
+*Label= will be the name of your line*
+
+![These People Went a Little Crazy With the Label Maker - Label Everything |  Guff](https://cdn.guff.com/site_1/media/20000/19031/thumbnails/fb1_6a99eaed05ac9d5f956c3e32.jpg)
+
+*`ax.plot` will plot your points/lines*
+
+## Step Six!
+
+Finishing touches!
+
+![Bibitty Bobittiy Boo!](https://45.media.tumblr.com/tumblr_lz56maEBTj1qdmlfso1_250.gif)
+
+   ```python 
+   ax.set_xlabel('Time') #adds X axis label
+   ax.set_ylabel('Years')#add Y axis label
+   ax.set_title("Chicken Little's Plot") #adds a title
+   ax.legend() #adds a key of the graph
+   ```
+
+You could possibly get someting that looks like this...
+**![](https://lh3.googleusercontent.com/x7NafZjPviM76xz95YQKc4L8kW4zC3B5_zp4wE7Qv4FKWVX-6ZoB_4qtT4WbA1er9soAcMHZeBNgJgLMttFnr1hiXOkWxLQpBXI3uQQyRd7wOElTYLWZtczqgTvmWvmdOEzmnqMTECg)**
+
+
+*If you are having any issues make sure to check your top right corner to check if the "kernel" is on and is set to Python 3*
+**![](https://lh6.googleusercontent.com/PJUhbx9vkJDp955g7Azwv_j_EyXGC-_YA9NSED74DU1XbTSxuDUrPfXN4S1b3TY-cjtqaSwUJv6Cnat6Zqko3WQBZ9oSZwncGlOL9EckE6zTwOLVuJ3F9CKQrO8INgII254BJiNcazc)**
+ 
+
+# If you are having any issues try copying the example code!
+
+**All Example Code:**
+
+   ```python 
+   import pandas as pd
+   import matplotlib.pyplot as plt
+   import numpy as np
+   
+   x=np.linspace(0,2,100)
+   fig,ax= plt.subplots()
+   ax.plot(x,x,label='Sleep')
+   ax.plot(x,x**2,label='Worry if sky is falling...')
+   ax.plot(x,x**3,label='Be a icon')
+   ax.set_xlabel('Time') #adds X axis label
+   ax.set_ylabel('Years')#add Y axis lable
+   ax.set_title("Chicken Little's Plot") #adds a title
+   ax.legend() #adds a key of the graph
+```
```

Comment:
You should have screenshots throughout of what your graph should look like if you’ve followed the code correctly